python/GafferOFXUITest/DocumentationTest.py

```
@@ -0,0 +1,54 @@
+#  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+#  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+#  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+#  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+##########################################################################
+
+import GafferUITest
+
+import GafferImage
+import GafferOFX
+import GafferOFXUI
```

Ruff (0.8.2)

`GafferOFXUI` imported but unused

Remove unused import: `GafferOFXUI`

(F401)